packages/generate-cnpj/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@brazilian-utils/generate-cnpj",
+  "description": "Generates a valid CNPJ",
+  "version": "0.1.5",
```

Comment:
Change version to `0.1.0`

Reply:
Changed in 51f252a58ddcde85e0ffbd5b318697658cb68418